Description: Improve printf syntax to work with hardened build options
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2012-12-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cross/plugins/Squeak3D/b3dMain.c
+++ b/Cross/plugins/Squeak3D/b3dMain.c
@@ -63,7 +63,7 @@
 /*************************************************************/
 
 void b3dAbort(char *msg){
-	printf(msg);
+	printf("%s", msg);
 	exit(-1);
 }
 
